benchmarking/data/real_data/cosc499_s2023_provider/providers.py

```diff
@@ -0,0 +1,57 @@
+from os import path
+from typing import List
+
+import numpy as np
+
+from api.models.student import Student, StudentSerializer
+from api.models.team import Team, TeamSerializer
+from benchmarking.data.interfaces import (
+    StudentProvider,
+    InitialTeamsProvider,
+)
+import json
+
+
+class COSC499S2023InitialTeamsProvider(InitialTeamsProvider):
+    def get(self) -> List[Team]:
+        """
+        Returns the teams that were created for the W2021T2 COSC341 class
```

Comment:
Wait like this returns the teams fr? The initial teams provider is for returning like shells of teams (I realized I gave the get() method the wrong return type)

Reply:
like for setting up the project requirements of each team shell and stuff